crates/typst-cli/src/compile.rs

```diff
@@ -198,6 +198,41 @@ enum ImageExportFormat {
     Svg,
 }
 
+struct OutputTemplate;
```

Comment:
Why is this a type? If we want the encapsulation, the `impl` could be a module I guess.

Reply:
I used it for internal state but I longer needed the state. I'll move it to a module